### PR TITLE
Military heli spawns

### PR DESCRIPTION
**Pull request by thelonestander**
#### Summary

SUMMARY: [Content] "Changed military helicopters spawns"

#### Purpose of change

I noticed in game that a lot of the military helicopters in game don't actually have spawn points. So I changed that. 

#### Describe the solution

I added military helicopter spawns to the military base, and the military helipads respectively.  

#### Describe alternatives you've considered

Would have to wait to add more spawns, but other than debug spawning there were no spawns. 

#### Testing

Tested in game by spawning a large number of military helipads and a Blackhawk spawned! 

#### Additional context

May have to adjust the X, and Y depending on how other helicopters spawn as they are quite big.

### CODE REVIEW

**Comment by silverwing235** [comment]
Slight goof here - "military helicopters pawns"?! This is currently a zombie apocalypse survival sim, not chess! :)

**Comment by thelonestander** [comment]
> Slight goof here - "military helicopters pawns"?! This is currently a zombie apocalypse survival sim, not chess! :)

Ahahaha changed it 🗡️ was so used to Rimworld

File: data/json/mapgen/helipad.json [hunk 1]
```diff
@@ -125,7 +125,8 @@
         { "vehicle": "flatbed_truck", "x": 11, "y": 4, "chance": 25, "rotation": 270, "status": -1 },
         { "vehicle": "SmallFlier3_Wreck", "x": 9, "y": 42, "chance": 10, "rotation": 90, "status": -1 },
         { "vehicle": "Smallflier1_Wreck", "x": 18, "y": 34, "chance": 10, "rotation": 270, "status": -1 },
-        { "vehicle": "SmallFlier2_Wreck", "x": 39, "y": 42, "chance": 10, "rotation": 90, "status": -1 }
+        { "vehicle": "mil_helicopters", "x": 39, "y": 42, "chance": 10, "rotation": 90, "status": -1 }
+
```
Comment (chaosvolt) [inline data/json/mapgen/helipad.json:129]: Delete this extraneous newline here and that should fix the JSON style error the buildbot is complaining about.

Comment (thelonestander) [reply data/json/mapgen/helipad.json]: Fixed :)

File: data/json/mapgen/helipad.json [hunk 2]
```diff
@@ -125,7 +125,7 @@
         { "vehicle": "flatbed_truck", "x": 11, "y": 4, "chance": 25, "rotation": 270, "status": -1 },
         { "vehicle": "SmallFlier3_Wreck", "x": 9, "y": 42, "chance": 10, "rotation": 90, "status": -1 },
         { "vehicle": "Smallflier1_Wreck", "x": 18, "y": 34, "chance": 10, "rotation": 270, "status": -1 },
-        { "vehicle": "SmallFlier2_Wreck", "x": 39, "y": 42, "chance": 10, "rotation": 90, "status": -1 }
+        { "vehicle": "mil_helicopters", "x": 39, "y": 42, "chance": 10, "rotation": 90, "status": -1 }
```
Comment (chaosvolt) [inline data/json/mapgen/helipad.json:128]: Couple spaces left on the end here somehow.

**Pull request merged**